tests/lib/cmsis_dsp/filtering/Kconfig

```diff
@@ -0,0 +1,13 @@
+# Copyright (c) 2019 Stephanos Ioannidis <root@stephanos.io>
+# SPDX-License-Identifier: Apache-2.0
+
+config CMSIS_DSP_TEST_FILTERING_BIQUAD
+	bool "Test: Filtering BIQUAD"
+
+config CMSIS_DSP_TEST_FILTERING_FIR
+	bool "Test: Filtering FIR"
+
+config CMSIS_DSP_TEST_FILTERING_MISC
+	bool "Test: Filtering MISC"
+
+source "$ZEPHYR_BASE/Kconfig"
```

Comment:
Can remove `$ZEPHYR_BASE` and do `source "Kconfig"`. The `srctree` environment variable will already be set to the Zephyr root.

Better to do `$(ZEPHYR_BASE)` btw, which uses the Kconfig preprocessor. `$ZEPHYR` is an older legacy syntax. Kconfiglib only supports it to be compatible with old Linux kernels.

Reply:
Done